presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaInternalFieldDescription.java

```diff
@@ -75,7 +77,12 @@
     /**
      * <tt>_key_length</tt> - length in bytes of the key.
      */
-    KEY_LENGTH_FIELD("_key_length", BigintType.BIGINT, "Total number of key bytes");
+    KEY_LENGTH_FIELD("_key_length", BigintType.BIGINT, "Total number of key bytes"),
+
+    /**
+     * <tt>_timestamp</tt> - offset timestamp
+     */
+    OFFSET_TIMESTAMP_FIELD("_timestamp", TimestampType.createTimestampType(DEFAULT_PRECISION), "Offset Timestamp");
```

Comment:
adding `OFFSET_TIMESTAMP_FIELD` could be a separate commit.

Reply:
Thanks. will resolve that.